### PR TITLE
A spooky Haiku

### PR DESCRIPTION
**Pull request by ChanceMartin4**
Perfect for Halloween!

### CODE REVIEW

File: _haikus/chance_haiku.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Chance's Haiku
+author:Chance
+---
+
+Welcome to the cloud<br>
+It is now your home forever<br>
```
Comment (ltagliaferri) [inline _haikus/chance_haiku.md]: This line is one syllable too long, you can amend it like this, for example:

```
It's now your home forever<br>
```

File: _haikus/chance_haiku.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Chance's Haiku
+author:Chance
```
Comment (ltagliaferri) [inline _haikus/chance_haiku.md]: Add a space between the colon and your name, like so:

```
author: Chance
```

Comment (ChanceMartin4) [reply _haikus/chance_haiku.md]: Fixed

File: _haikus/chance_haiku.md (hunk 1 above)
Comment (ChanceMartin4) [reply _haikus/chance_haiku.md]: Oh woops, my mistake.

**Pull request merged**